.github/workflows/ci-examples-app.yml

```diff
@@ -73,10 +73,19 @@ jobs:
         key: pypi_wheels
 
     - name: Install Lightning package & dependencies
+      run: |
+        # do not use `-e` because it will make both packages available since it adds `src` to `sys.path` automatically
+        pip install .[app-dev] -U -f ${TORCH_URL} -f ${PYPI_CACHE_DIR} --prefer-binary
+        pip list
+      if: ${{ matrix.pkg-name == 'lightning' }}
+    - name: Install Lightning package & dependencies standalone
       run: |
         # do not use `-e` because it will make both packages available since it adds `src` to `sys.path` automatically
         pip install .[dev] -U -f ${TORCH_URL} -f ${PYPI_CACHE_DIR} --prefer-binary
```

Comment:
```suggestion
      run: |
        extra=$(python -c "print({'app': 'app-'}.get('${{ matrix.pkg-name }}', ''))")
        # do not use `-e` because it will make both packages available since it adds `src` to `sys.path` automatically
        pip install ".[${extra}dev]" -U -f ${TORCH_URL} -f ${PYPI_CACHE_DIR} --prefer-binary
```